Source/Documentation/Manual/driving.rst

```diff
@@ -1606,8 +1606,8 @@ key assignment list shown by pressing ``<F1>``.
 In addition, Open Rails offers functionality similar to the time 
 acceleration switch for MSTS. 
 
-Use ``<Alt+PgUp>`` or ``<Alt+PgDn>`` keys to increase or decrease the 
-speed of the game clock.
+Use ``<Ctrl+Alt+PgUp>`` or ``<Ctrl+Alt+PgDn>`` keys to increase or decrease the 
+speed of the game clock. ``<Ctrl+Alt+Home>`` resets the speed.
```

Comment:
You should say it is the 9, 3 and 7 keys from the numeric keypad in order to not confuse them with the other keys called PgUp, PgDn and Home.

Reply:
Modified.

Reply:
@Sharpe49 Actually it was designed for the main keys; it _also_ works on the numeric keypad, but only with Num Lock off